bin/base64

```diff
@@ -1,5 +1,19 @@
 #!/usr/bin/perl
 
+=begin metadata
+
+Name: base64
+Description: encode and decode base64 data
+Author: Michael Mikonos
+License: perl
```

Comment:
Change this to artistic2. Or, ask Michael what he wants.

We want to avoid the GPL for new things because that causes problems for some people and it doesn't really benefit this project. (I know you said you were a license compliance guy in a former life).

Some other programs in this repo use GPL, but that's what they were originally so that's how it is. I'd be inclined to replace those with new versions just to reduce the license count.